### PR TITLE
README: add ./generate.py instructions

### PR DESCRIPTION
**Pull request by lucaswerkmeister**
Since Twitter suspended @this_pigeon (:sob:), I figure these instructions are useful. (I’m not sure if the example message is ideal, though… perhaps we need to tone down the salt ;) )

### CODE REVIEW

**Comment by saleemrashid** [comment]
Thanks for the pull request! Since that functionality was originally added for debug purposes, it might be an idea to rewrite that to `argparse` and add a `-o/--output` and a `--open` option before documenting it?

I think the example message has a suitable quantity of salt :wink:

**Comment by lucaswerkmeister** [comment]
Good point, done. (I changed the `--open` to `--view` so that it could have a short option, is that okay?)

File: README.md [hunk 1]
```diff
@@ -50,3 +50,15 @@ export TWITTER_ACCESS_TOKEN_SECRET="<insert-access-token-secret>"
 ```sh
 ./main.py
 ```
+
+### Non-bot usage
+
+You can also directly generate individual images locally:
+
+```sh
+./generate.py Twitter admins: is @this_pigeon bad?
```
Comment (saleemrashid) [inline README.md]: Could you quote this? (as some shells will expand the `?` as a glob)

File: generate.py [hunk 2]
```diff
@@ -142,13 +142,21 @@ def generate_image(text, fp):
 
 
 if __name__ == "__main__":
+    import argparse
     import sys
     import tempfile
 
-    fd, filename = tempfile.mkstemp(".png")
-    text = " ".join(sys.argv[1:])
+    parser = argparse.ArgumentParser(description='Generate an Is This A Pigeon? image.', epilog='Example: ./generate.py me: Is \\"a single example\\" good documentation?')
```
Comment (saleemrashid) [inline generate.py]: Not sure about the `description` and `epilog`. Could we leave this blank for now?

**Review by saleemrashid — CHANGES_REQUESTED** [review]
Would it be possible to make those two changes and switch all the single quotes to double quotes, for consistency?

Thanks for all the work!

**Comment by lucaswerkmeister** [comment]
Sure, no problem! :)

**Review by saleemrashid — APPROVED** [review]

**Comment by saleemrashid** [comment]
Thanks!

**Comment by lucaswerkmeister** [comment]
No problem, thank you for writing this :)

**Pull request merged**